core/build.gradle.kts

```diff
@@ -318,11 +298,13 @@ val downloadWindowsZonesMapping by tasks.registering {
         xmlDoc.documentElement.normalize()
         val mapZones = xmlDoc.getElementsByTagName("mapZone")
         val mapping = linkedMapOf<String, String>()
+        mapping["UTC"] = "UTC"
         for (i in 0 until mapZones.length) {
             val mapZone = mapZones.item(i)
             val windowsName = mapZone.attributes.getNamedItem("other").nodeValue
             val usualNames = mapZone.attributes.getNamedItem("type").nodeValue
```

Comment:
I've noticed that each `mapZone` also contains `territory` attribute. Can it potentially be used to make Windows->IANA mapping less ambiguous?

Reply:
I don't see how. AFAIK, the only pieces of information about the current time zone are those in this picture: https://static1.makeuseofimages.com/wordpress/wp-content/uploads/2022/11/Change-Time-Zone-Using-Registry-Editor.jpg?q=50&fit=crop&w=1500&dpr=1.5 I don't see any territorial markings here.